### PR TITLE
Update install-the-warehouse-picking-shipment-feature.md

### PR DESCRIPTION
**Pull request by AlexSlawinski**
## PR Description

TBD

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: _includes/pbc/all/install-features/202304.0/install-the-warehouse-picking-shipment-feature.md [hunk 1]
```diff
@@ -14,8 +14,8 @@ To start feature integration, integrate the required features:
 
 | NAME              | VERSION          | INTEGRATION GUIDE                                                                                                                              |
 |-------------------|------------------|------------------------------------------------------------------------------------------------------------------------------------------------|
-| Warehouse Picking | {{site.version}} | [Warehouse Picking feature integration](/docs/scos/dev/feature-integration-guides/{{site.version}}/install-the-warehouse-picking-feature.html) |
-| Shipment          | {{site.version}} | [Shipment feature integration](/docs/pbc/all/carrier-management/{{page.version}}/install-and-upgrade/install-the-shipment-feature.html)        |
+| Warehouse Picking | {{site.version}} | [Warehouse Picking feature integration](/docs/scos/dev/feature-integration-guides/202304.0/install-the-warehouse-picking-feature.html) |
```
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202304.0/install-the-warehouse-picking-shipment-feature.md:17]: According to our docs, links should look like this:
```suggestion
| Warehouse Picking | {{page.version}} | [Warehouse Picking feature integration](/docs/scos/dev/feature-integration-guides/{{page.version}}/install-the-warehouse-picking-feature.html) |
```

File: _includes/pbc/all/install-features/202304.0/install-the-warehouse-picking-shipment-feature.md [hunk 2]
```diff
@@ -14,8 +14,8 @@ To start feature integration, integrate the required features:
...
+| Shipment          | {{site.version}} | [Shipment feature integration](/docs/pbc/all/carrier-management/202212.0/base-shop/install-and-upgrade/install-features/install-the-shipment-feature.html)        |
```
Comment (VadymSachenko) [inline _includes/pbc/all/install-features/202304.0/install-the-warehouse-picking-shipment-feature.md:18]: same here:
```suggestion
| Shipment          | {{page.version}} | [Shipment feature integration](/docs/pbc/all/carrier-management/{{page.version}}/base-shop/install-and-upgrade/install-the-shipment-feature.html)        |
```

File: _includes/pbc/all/install-features/202304.0/install-the-warehouse-picking-shipment-feature.md (hunk 1 above)
Comment (lenadoc) [inline _includes/pbc/all/install-features/202304.0/install-the-warehouse-picking-shipment-feature.md:17]: We should not hardcode the version number. Instead {{site.version}} or {{page.version}} variables are to be used. See the difference here: https://docs.spryker.com/docs/scos/user/intro-to-spryker/contribute-to-the-documentation/style-formatting-general-rules.html#working-with-versions Try using {{page.version}} here

File: _includes/pbc/all/install-features/202304.0/install-the-warehouse-picking-shipment-feature.md (hunk 2 above)
Comment (lenadoc) [inline _includes/pbc/all/install-features/202304.0/install-the-warehouse-picking-shipment-feature.md:18]: same here, don't hardcode the version number, use the above-mentioned variables. After you make the fixes, make sure to build the website locally to make sure your changes fixed the issues

**Review by lenadoc — CHANGES_REQUESTED** [review]

**Review by VadymSachenko — DISMISSED** [review]

**Pull request merged**